CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Add macOS ARM64 platform to the compatibility testing suite. (#5577)
 
+### Changed
+
+- `NewMember` and `NewKeyValueProperty` in `go.opentelemetry.io/otel/baggage` allow percent-encoded UTF-8 string in key. (#5132)
```

Comment:
Why are you not using the same key validation for `NewMember` and `NewKeyValueProperty` as for other functions?

Reply:
You are right. I should use `validateValue` (but also verify the length) to verify the key for `NewMember` and `NewKeyValueProperty`. Otherwise, it won't accept percent-encoded key.

Reply:
But we don't provide a propagator that would publish the percent-encoded key. It now confuses me how we should deal with `NewMember` and `NewKeyValueProperty`. Maybe we should keep their behavior intact? (the `Parse` method does not use `NewMember` and `NewKeyValueProperty`)

Reply:
> Maybe we should keep their behavior intact?

This is what I would suggest as well

Reply:
Done